src/lib/unzip.ts

```diff
@@ -30,7 +30,7 @@ export async function attemptUnzip(pathInput: string): Promise<MarkdownFileScanR
       const newWorkingDir = path.join(outputDir, path.basename(pathInput, '.zip'));
 
       return { pathInput: newWorkingDir, zipped: true as const, unzippedDir: newWorkingDir };
-    } catch (e) {
+    } catch (_e) {
```

Comment:
Biome flags when we don't use variables but since we can't do `} catch () {` it says to prefix an unused variable with an underscore.